### PR TITLE
 it's the Cash rsjester project ,work with index.html styles.css and script.js

### PR DESCRIPTION
**Pull request by fatiabdulahi**
This this the Chash register project that registers cache price and check if the price is less than value or it is fix value or more than price.

### CODE REVIEW

**Comment by fatiabdulahi** [comment]
@sharifarezaie hi dear sharifa please cheack my project and shear your opinian

**Comment by fatiabdulahi** [comment]
@ahzamir hello sir I copleated my project please review it thank you

**Comment by Sharifarezaie** [comment]
Hi Fatima jan  thanks for  mention
Me l think  you're  project  have not problems  when  thecher  review  your
Project  you can merge  pul requests
Good 👍 job 👏

**Comment by ahzamir** [comment]
@fatiabdulahi 

- There are some issues in your project so when I type a number or a price and click purchase your app crashes and not working.

**Comment by fatiabdulahi** [comment]
> @fatiabdulahi
> 
> * There are some issues in your project so when I type a number or a price and click purchase your app crashes and not working.

@thank you sir for your time

**Comment by fatiabdulahi** [comment]
@ahzamir hello dear Sir I have solved my project issue please review it again.

File: styles.css
Comment (ahzamir) [inline styles.css:1]: - Your design in mobile version is not responsive please solve the issue and then directly merge your branch:
image

**Review by ahzamir — APPROVED** [review]
### Hi @fatiabdulahi 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- Your design is not responsive please solve the issue and then directly merge your branch.

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text with a screenshot.

- Well Done great design Implemented. 🎯🎉👍
image

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

WinkThumbsUpGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Cash Register project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your repo has a professional README.md file
✔️ Your partner has reviewed your repo
✔️Your design is responsive

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me @ahzamir in your question so I can receive the notification.**

**Pull request merged**